proposals/low-level-struct-improvements.md

```diff
@@ -136,21 +136,25 @@ The rules we define for `ref` fields must ensure the `Span<T>` constructor prope
 
 <a name="ref-fields-escape"></a>
 
-To accomplish this the span safety rules will be changed as follows. First for the constructor of a `ref struct` that directly contains a `ref field`: 
-- If the constructor is annotated with `[RefFieldEscapes]` then `ref` fields can be initialized with any value that has *ref-safe-to-escape* to the *calling method*
-- Else `ref` fields can only be initialized with known heap values.
+To accomplish this two new annotations will be introduced to help control how arguments influence lifetime of method calls: `[RefFieldsEscape]` and `[DoesNotEscape]`. The annotation `[RefFieldsEscape]` when applied to a `ref` parameter signifies that it can be captured as a `ref` field in a returned `ref struct`.
```

Comment:
>RefFieldsEscape [](http://example.com/codeflow?start=126&length=15)

Consider s/[RefFieldsEscape]/[RefFieldEscapes]/ since there are currently a mix of the two.